engine/table/src/main/java/io/deephaven/engine/table/impl/updateby/UpdateBy.java

```diff
@@ -358,12 +358,25 @@ private void onError(@NotNull final Exception error) {
          * {@code completedAction} when the work is complete
          */
         private void computeCachedColumnRowsets(final Runnable resumeAction) {
+            // Determine which windows need to be computed.
```

Comment:
I think `processUpdate` could return earlier if there are no dirty buckets.
This could happen if there were only shifts, I think. Also if there were only mods to rows that influenced nothing.